pkgs/by-name/du/duckstation/package.nix

```diff
@@ -71,21 +71,22 @@ stdenv.mkDerivation (finalAttrs: {
   ]
   ++ cubeb.passthru.backendLibs;
 
-  strictDeps = true;
-
   cmakeFlags = [
     (lib.cmakeBool "BUILD_TESTS" true)
   ];
 
-  doInstallCheck = true;
-
-  installCheckPhase = ''
-    runHook preCheck
+  qtWrapperArgs =
```

Comment:
There is no concensus even in https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md about where to put this argument. I think this commit should be dropped.